avaframe/com1DFA/com1DFA.py

```diff
@@ -462,10 +461,23 @@ def prepareInputData(inputSimFiles, cfg):
     # load data
     entResInfo = inputSimFiles['entResInfo'].copy()
     relFile = inputSimFiles['releaseScenario']
+    relThFile = inputSimFiles['relThFile']
 
     # get dem information
     demOri = IOf.readRaster(inputSimFiles['demFile'], noDataToNan=True)
 
+    # read data from relThFile
+    if relThFile != '':
+        relThField = IOf.readRaster(relThFile)
+        relThFieldData = relThField['rasterData']
+        if demOri['header']['ncols'] != relThField['header']['ncols'] or demOri['header']['ncols'] != relThField['header']['ncols']:
+            message = ('Release thickness field read from %s does not match the number of rows \
+                       and columns of the dem' % inputSimFiles['relThFile'])
+            log.error(message)
+            raise AssertionError(message)
```

Comment:
I think this is an important error to check for via pytest!

Reply:
included in pytest